zerver/actions/message_edit.py

```diff
@@ -804,6 +804,7 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
             # unless the topic has thousands of messages of history.
             assert stream_being_edited.recipient_id is not None
             unmoved_messages = messages_for_topic(
+                stream_being_edited.realm_id,
```

Comment:
I think there's still a `realm.id` in this scope -- part of the same function as the previous hunk.